packages/components-react/projects/nextjs/app/modal-route-change/layout.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+import { type PropsWithChildren, useCallback, useState } from 'react';
+import { PHeading, PModal, PText, PLink, PButtonGroup, PButton } from '@porsche-design-system/components-react/ssr';
+import Link from 'next/link';
+
+export default function ModalRouteChangeLayout({ children }: PropsWithChildren<{}>): JSX.Element {
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+  const onOpen = useCallback(() => {
+    setIsModalOpen(true);
+  }, []);
+  const onDismiss = useCallback(() => {
+    setIsModalOpen(false);
+  }, []);
+
+  const style = `
+    #app {
+      transform: none;
+    }
+  `;
+
+  return (
+    <section>
+      <PHeading>Welcome to Overview</PHeading>
+      <PLink>
+        <Link href="/">Back to Home</Link>
+      </PLink>
+      <div style={{ display: 'grid', gridTemplateColumns: '1fr 1fr' }}>
+        <div
+          style={{
+            height: '1000px',
+            background: 'deeppink',
+          }}
+        ></div>
+        <div
+          style={{
+            height: '1000px',
+            background: 'darkcyan',
+          }}
+        ></div>
+      </div>
+      <PLink>
+        <Link href="/modal-route-change/open" scroll={false}>
```

Comment:
I guess `scroll: false` does the entire trick? Might be good to comment this so it is clear right away.

Reply:
commented it